selection_sort.go

```diff
@@ -0,0 +1,16 @@
+package sort
```

Comment:
The package declaration is correct, but it's good practice to include a comment at the top of the file explaining the purpose of the package and any important information about its usage or dependencies.